### ISSUE TITLE
Unicode-aware identifier in calculator example does not work

### ISSUE THREAD

**Issue by verri** [foonathan/lexy#66]
I believe utf-8 identifiers like 'é' should work.

Example of input:

```
> é=10
error: while parsing stmt
     │
   1 │ ⟨0xC3⟩⟨0xA9⟩=10
     │       ^^^^^^ expected ';'

= 0
```

**Comment by verri** [comment]
Now I get it, I had to change

```cpp
lexy_ext::shel<>
```

to

```cpp
lexy_ext::shell<lexy_ext::default_prompt<lexy::utf8_encoding>>
```

Is it the intended behavior?

**Comment by foonathan** [comment]
> Is it the intended behavior?

Sort of, by default you don't get Unicode input, so you need to change the input. However, it shouldn't have compiled in the first place. This was a consequence of the fix for #53, which actually causes buggy behavior.

**Comment by foonathan** [comment]
It's now fixed.
